.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: bash
+sudo: required
```

Comment:
we need to install the shellcheck package. this isn't yet whitelisted for the containerinfra so we need sudo :(